### DISCUSSION TITLE
Radio Link R12 DS with IRX4 no binding

### DISCUSSION THREAD

**Discussion by Quaxpilot** [pascallanger/DIY-Multiprotocol-TX-Module#829] [Q&A]
Hi from Germany,

i bought a RadioLink R12DS Receiver and wanted to bind it with the IRX4 Module in my Taranis X9. Diddn´t work. I updated everything no way. Any Ideas or simply no Protocol available?

Thanks Thorsten

**Comment by pascallanger** [comment]
Check the documentation: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#radiolink---74
